benchmarks/StreamThroughput.cpp

```diff
@@ -22,39 +23,7 @@ using namespace rsocket;
 DEFINE_string(host, "localhost", "host to connect to");
 DEFINE_int32(port, 0, "host:port to connect to");
 
-class BM_Subscription : public yarpl::flowable::Subscription {
- public:
-  explicit BM_Subscription(
-      yarpl::Reference<yarpl::flowable::Subscriber<Payload>> subscriber,
-      size_t length)
-      : subscriber_(std::move(subscriber)),
-        data_(length, 'a'),
-        cancelled_(false) {}
-
- private:
-  void request(int64_t n) noexcept override {
-    VLOG(3) << "requested=" << n << " currentElem=" << currentElem_;
-
-    for (int64_t i = 0; i < n; i++) {
-      if (cancelled_) {
-        VLOG(3) << "emission stopped by cancellation";
-        return;
-      }
-      subscriber_->onNext(Payload(data_));
-      currentElem_++;
-    }
-  }
-
-  void cancel() noexcept override {
-    VLOG(3) << "cancellation received";
-    cancelled_ = true;
-  }
-
-  yarpl::Reference<yarpl::flowable::Subscriber<Payload>> subscriber_;
-  std::string data_;
-  size_t currentElem_ = 0;
-  std::atomic_bool cancelled_;
-};
+DEFINE_int32(items, 1000000, "number of items in stream");
```

Comment:
Why was BENCHMARKPARAM not working for Travis?
It is nice to have various numbers for this parameter to observe the complexity of the functionalities, like if we double the size or quadruple it how much the spent time increases, linear or not, or what's the multiplier for liner O(k.n) etc.